src/styles/main.scss

```
@@ -1,3 +1,59 @@
+@import "./utils/variables";
+
 body {
-  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding;
+  width: $cellSize * 7 + 100 + ($gap * 6);
+  gap: $gap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $cellSize;
+    height: $cellSize;
+    background-color: $cell-background-color;
+    border: 1px solid black;
+    transition: ease-in-out 0.3s;
```

list all properties like is in example here: https://www.w3schools.com/css/css3_transitions.asp
```suggestion
    transition: background-color 0.3s ease-in-out, transform 0.3s ease-in-out;
```